internal/installcfg/installcfg.go

```diff
@@ -144,15 +143,7 @@ func (i *installConfigBuilder) countHostsByRole(cluster *common.Cluster, role mo
 }
 
 func (i *installConfigBuilder) getBMHName(host *models.Host, masterIdx, workerIdx *int) string {
-	prefix := "openshift-master-"
-	index := masterIdx
-	if host.Role == models.HostRoleWorker {
-		prefix = "openshift-worker-"
-		index = workerIdx
-	}
-	name := prefix + strconv.Itoa(*index)
-	*index = *index + 1
-	return name
+	return hostutil.GetHostnameForMsg(host)
```

Comment:
I think instead of this, you can directly do 
```
hostName := hostutil.GetHostnameForMsg(host)
i.log.Infof("host name is %s", hostName)
hosts[yamlHostIdx].Name = hostName
```
here https://github.com/openshift/assisted-service/pull/1992/files#diff-f9d8c8224b172499b34cb44479e6e3606f19036989fa80449d085d1f3780e6eeR286-R287
and remove `getBMHName` function alltogether.

Reply:
Good catch, thanks! 👍🏻